test/hummingbot/connector/exchange/ascend_ex/test_ascend_ex_utils.py

```diff
@@ -51,11 +51,15 @@ def test_get_ws_url_private(self):
         self.assertEqual(f"wss://ascendex.com/{account_id}/api/pro/v1", url)
 
     def test_get_ms_timestamp(self):
-        with mock.patch('hummingbot.connector.exchange.ascend_ex.ascend_ex_utils.get_tracking_nonce_low_res') as get_tracking_nonce_low_res_mock:
-            get_tracking_nonce_low_res_mock.return_value = 123456789
+        with mock.patch('hummingbot.connector.exchange.ascend_ex.ascend_ex_utils._time') as get_tracking_nonce_low_res_mock:
```

Comment:
Maybe it will be good to be more consistent with our unit tests and use `@patch` instead of using a `with` statement. Also since we're no longer using `get_tracking_nonce_low_res`, lets change that mock name as well.